pos_mail_receipt/static/src/css/print.css

```diff
@@ -0,0 +1,129 @@
+/* This is a copy of the pos receipt screen CSS
+ * This allows to use it in the backend */
+.pos .receipt-screen .centered-content .button {
+    line-height: 40px;
+    padding: 3px 13px;
+    font-size: 20px;
+    text-align: center;
+    background: rgb(230, 230, 230);
+    margin: 16px;
+    margin-bottom: 0px;
+    border-radius: 3px;
+    border: solid 1px rgb(209, 209, 209);
+    cursor: pointer;
+}
+
+.pos .pos-receipt-container {
+    font-size: 0.75em;
+    text-align: center;
+    direction: ltr;
+}
+
+.pos .pos-sale-ticket {
+    text-align: left;
+    width: 300px;
+    background-color: white;
+    margin: 20px;
+    padding: 15px;
+    font-size: 14px;
+    padding-bottom:30px;
+    display: inline-block;
+    font-family: "Inconsolata";
+    border: solid 1px rgb(220,220,220);
+    border-radius: 3px;
+    overflow: hidden;
+}
+.pos .pos-sale-ticket pre{
+    font-family: "Inconsolata";
+}
+.pos .pos-sale-ticket .emph{
+    font-size: 20px;
+    margin:5px;
+}
+.pos .pos-sale-ticket table {
+    width: 100%;
+    border: 0;
+    table-layout: fixed;
+}
+.pos .pos-sale-ticket table td {
+    border: 0;
+    word-wrap: break-word;
+}
+
+@page {
```

Comment:
All this part will alter all the pdf rendered via ``web.report_assets_pdf``. Don't you think ? 
If yes it should be avoided.

Reply:
This comment is still pending AFAICS.

@legalsylvain has a point here, this css is not scoped so it'll be applied for any report and that's not good.

Reply:
Done, I've encapsulated the CSS